webcompat/templates/cssfixme.html

```diff
@@ -9,10 +9,11 @@
       <p>Paste CSS in this box to see what needs fixing:</p>
 
       <textarea style="width:100%;height:20rem;"></textarea><br/>
-      <button type="button" onclick="doTheBigStyleFixing(document.getElementsByTagName('textarea')[0].value)">Add standard equivalents for prefixed CSS</button>
+      <button type="button" id="btn_do_fixup">Add standard equivalents for prefixed CSS</button>
       <label>
         <input type="checkbox" name="compact">Compact output (whitespace only after closing brace)</label>
       <pre id="fixedcss"></pre>
+      <script src="/js/cssfixme.min.js"></script>
```

Comment:
Traditionally we've done the following for scripts that have build processes:

```
{%- block extrascripts -%}
{%- if config.PRODUCTION or config.DEVELOPMENT -%}
<script src="{{ url_for('static', filename='js/diagnose.min.js')|bust_cache }}"></script>
{%- else -%}
<script src="{{ url_for('static', filename='js/lib/models/label-list.js') }}"></script>
<script src="{{ url_for('static', filename='js/lib/labels.js') }}"></script>
<script src="{{ url_for('static', filename='js/lib/models/issue.js') }}"></script>
<script src="{{ url_for('static', filename='js/lib/diagnose.js') }}"></script>
{%- endif -%}
{%- endblock %}
```

This way we're doing cache busting and we can more easily debug in dev environments. Any reason to not do something similar here?